### ISSUE TITLE
Add alias button has weird colors in light mode

### ISSUE THREAD

**Issue by DerMolly** [hedgedoc/hedgedoc#5087]
### Which part of the project should be enhanced?

alias `+` button

### Is your enhancement request related to a problem? Please describe.

The alias `+` button is a bit hard to distinguish in light mode. Both clickable and deactivated look very similiar.

### Describe the solution you'd like

Make the contrast better and the state of the button more apparent.

### Describe alternatives you've considered

-

### Additional context

https://github.com/hedgedoc/hedgedoc/assets/3978662/8ecd4db3-6a85-477b-993e-205e55b9c2cf

**Comment by yamashush** [comment]
Hi @DerMolly @ErikMichelson

I'm thinking of changing it like below, what do you guys think?
I would be happy to receive your comments 😄 

- ➕ Add alias button
  - The operation that users most want to do on this modal.
  - → use primary variant 🟢 
- ⭐ Set primary alias button
  - The operation that users want to do next on this modal.
  - → use secondary variant ⚪️
  - And add `Primary Alias` text to current primary alias because users don't know what the star means.
- ❌ Remove alias button
  - Removing is dangerous operation. (A little dangerous in the case of this modal)
  - → use danger variant 🔴  

![image](https://github.com/hedgedoc/hedgedoc/assets/38120991/8df8b2cf-3dd2-43ed-a54b-15a0a36c1de9)

![image](https://github.com/hedgedoc/hedgedoc/assets/38120991/d2cb3895-aae4-46f0-b89a-059823fd140b)

**Comment by mrdrogdrog** [comment]
I like the changes in general. The only thing i'm not really a fan of is the text in the favorite-button. It doesn't really provide more useful information and breaks the layout. I would prefer if the button just gets disabled or not even rendered.

**Comment by DerMolly** [comment]
> I like the changes in general. The only thing i'm not really a fan of is the text in the favorite-button. It doesn't really provide more useful information and breaks the layout. I would prefer an alternative that doesn't blow up the button so much. Like the button getting disabled or not even rendered.

Maybe it would be best to add the "Primary Alias" after the name instead of the button?

**Comment by mrdrogdrog** [comment]
That's also possible, but then I would use a [badge](https://react-bootstrap.netlify.app/docs/components/badge/)

**Comment by DerMolly** [comment]
> That's also possible, but then I would use a [badge](https://react-bootstrap.netlify.app/docs/components/badge/) AND disable / hide the button.

Yeah, but we need to make sure that this badge is clearly distinguishable from a button even if it's directly after the name and not at the end where the button normally be…

**Comment by yamashush** [comment]
> The only thing i'm not really a fan of is the text in the favorite-button.

I feel you very well 👍 

How is to use badges like this?

- Place informations on the left ◀️ 
  - list texts
  - badges
    - to distinguish from buttons
- Place user operation elements on the right ▶️ 
  - buttons

![image](https://github.com/hedgedoc/hedgedoc/assets/38120991/f5e69ff1-169e-4ea2-8a30-7861c943cc7a)

**Comment by mrdrogdrog** [comment]
That's how I had it in mind.

**Comment by yamashush** [comment]
Thank you, I'll try implementing it.

**Comment by ErikMichelson** [comment]
Thank you, this looks definitely better!

**Comment by ErikMichelson** [comment]
Already implemented in #5386 thanks to @yamashush

**Comment by yamashush** [comment]
@ErikMichelson I appreciate it too. Thank you for your review.
